### DISCUSSION TITLE
homme tool error

### DISCUSSION THREAD

**Discussion by gourikadam** [E3SM-Project/E3SM#7762] [E3SM model help]
Hi,

Configuring homme tool on unsupported platform using " cmake -C /scratch/um_cdac2/met_models/CDACP_Int/E3SM/components/homme/cmake/machineFiles/param-shivay-homme.cmake -DBUILD_HOMME_WITHOUT_PIOLIBRARY=True -BUILD_HOMME_PREQX_KOKKOS=Off -DPREQX_NP=4 -DHOMME_PROJID=XXXX0000 /scratch/um_cdac2/met_models/CDACP_Int/E3SM/components/homme/ make homme_tool"

it gives below error

 "CMake Error at CMakeLists.txt:463 (include):
  include could not find requested file:

    EkatFindKokkos

CMake Error at CMakeLists.txt:467 (include):
  include could not find requested file:

    EkatBuildKokkos"

and error "-- Configuring incomplete, errors occurred!"

**Comment by mt5555** [comment]
You might be missing various submodules (like ekat) in your E3SM checkout.   
try:  git submodule update  --recursive --init 

I noticed you have I/O disabled ( -DBUILD_HOMME_WITHOUT_PIOLIBRARY=True).   Assuming this is just to get past netcdf errors for now?   that will also have to be addressed before any of the utilities are useful.
